.github/workflows/main.yml

```diff
@@ -48,9 +48,9 @@ jobs:
         nix-shell \
           -E 'with import ./nixpkgs-pinned.nix {}; mkShell { buildInputs = [ hpack execline cabal2nix ]; }' \
           --run '
-            execline-cd yarn2nix redirfd -w 1 yarn2nix.nix cabal2nix . \
+            execline-cd yarn2nix redirfd -w 1 default.nix cabal2nix . \
             && \
-            execline-cd yarn-lock redirfd -w 1 yarn-lock.nix cabal2nix . \
+            execline-cd yarn-lock redirfd -w 1 default.nix cabal2nix . \
```

Comment:
not sure this change is good because these are not really callable with `nix-build`, so I don’t want to call them `default.nix`.

Reply:
I guess, yeah, but it makes tab completing `yarn2nix.cabal` less annoying which was my main motivation for this.

Reply:
Cool, then let’s revert this commit.